test/unit/constants.dart

```diff
@@ -193,3 +192,9 @@ final List<String> draft9SkippedTestFiles = [
   "uuid.json",
   "vocabulary.json",
 ]..addAll(commonSkippedTestFiles);
+
+final List<String> draft2019SkippedTests = [
+  'remote ref, containing refs itself : remote ref valid',
```

Comment:
This seems like something we should support, yeah?

Reply:
Chatted live - these tests reference schemas that include keywords we have not yet implemented. After we implement those, we'll come back and clear out this list.

Reply:
Definitely want these to pass before release. But fine in this PR